### PR TITLE
fix(ci): actually run ingest consumer kafka tests

### PR DESCRIPTION
**Pull request by joshuarli**
These have been xfailing for probably quite a long time since the normal backend tests do not run the kafka service:

```
XFAIL tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py::test_ingest_consumer_reads_from_topic_and_calls_celery_task[synchronous]
  reason: test requires SENTRY_KAFKA_HOSTS environment variable which is not set
XFAIL tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py::test_ingest_consumer_reads_from_topic_and_calls_celery_task[asynchronous]
  reason: test requires SENTRY_KAFKA_HOSTS environment variable which is not set
XFAIL tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py::test_ingest_topic_can_be_overridden
  reason: test requires SENTRY_KAFKA_HOSTS environment variable which is not set
XFAIL tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py::test_ingest_consumer_fails_when_not_autocreating_topics
  reason: test requires SENTRY_KAFKA_HOSTS environment variable which is not set
```

So, this moves them to test-snuba which does run kafka.

Not sure if they'll actually pass or not, lol

### CODE REVIEW

**Comment by joshuarli** [comment]
Okay, even though these are moved to snuba, they still xfail for the same reason. Gonna fix that.

**Comment by joshuarli** [comment]
Now that these kafka tests are actually running, this one doesn't work. Gonna xfail it for now and leave it to the ingest team.

```
_____________________ test_ingest_topic_can_be_overridden ______________________
tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py:187: in test_ingest_topic_can_be_overridden
    consumer = get_ingest_consumer(
src/sentry/ingest/ingest_consumer.py:433: in get_ingest_consumer
    return create_batching_kafka_consumer(
src/sentry/utils/kafka.py:77: in create_batching_kafka_consumer
    consumer = BatchingKafkaConsumer(
src/sentry/utils/batching_kafka_consumer.py:257: in __init__
    self.consumer = self.create_consumer(
src/sentry/utils/batching_kafka_consumer.py:312: in create_consumer
    create_topics(topics)
src/sentry/utils/batching_kafka_consumer.py:66: in create_topics
    cluster_names = {settings.KAFKA_TOPICS[topic]["cluster"] for topic in topics}
src/sentry/utils/batching_kafka_consumer.py:66: in <setcomp>
    cluster_names = {settings.KAFKA_TOPICS[topic]["cluster"] for topic in topics}
E   KeyError: 'ingest-events-new'
```

**Review by asottile-sentry — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -129,12 +129,18 @@ test-python-ci:
...
 	@echo ""
 
 test-snuba:
 	@echo "--> Running snuba tests"
-	pytest tests/snuba tests/sentry/eventstream/kafka tests/sentry/snuba tests/sentry/search/events -vv --cov . --cov-report="xml:.artifacts/snuba.coverage.xml" --junit-xml=".artifacts/snuba.junit.xml"
+	pytest tests/snuba \
+		tests/sentry/eventstream/kafka \
+		tests/sentry/snuba \
+		tests/sentry/search/events \
+		tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py \
```
Comment (jan-auer) [inline Makefile:142]: There's also `test-relay-integration` which can run Kafka etc, could we rather add it there? Relay and the ingest consumers are closer to each other -- in fact, Relay integration tests also partially cover ingest consumers already.

**Review by jan-auer — APPROVED** [review]
Thanks, one suggestion below. 

I'll have a look at `test_ingest_topic_can_be_overridden` in a follow-up.

File: Makefile (hunk 1 above)
Comment (joshuarli) [reply Makefile:142]: Oh whoops, I forgot I set this to automerge. that's a better place for it, yeah

**Pull request merged**